src/components/shared/data/skills.js

```diff
@@ -0,0 +1,340 @@
+
+                }
+            }
+        }
+    ],
+    "Turret": [
+        {
+            name: "Turret",
+            baseStats: {
+                Cooldown: "60s",
+            },
+            tiers: {
+                1: {
+
+                },
+                2: {
+
+                },
+                3: {
+
+                },
+                4: {
+
+                },
+                5: {
+
+                },
+                6: {
+
+                }
+            }
+        }
+    ],
+    "Hive": [
+        {
+            name: "Hive",
+            baseStats: {
+                Cooldown: "60s",
+            },
+            tiers: {
+                1: {
+
+                },
+                2: {
+
+                },
+                3: {
+
+                },
+                4: {
+
+                },
+                5: {
+
+                },
+                6: {
+
+                }
+            }
+        }
+    ],
+    "Chem Launcher": [
+        {
+            name: "Chem Launcher",
+            baseStats: {
+                Cooldown: "60s",
+            },
+            tiers: {
+                1: {
+
+                },
+                2: {
+
+                },
+                3: {
+
+                },
+                4: {
+
+                },
+                5: {
+
+                },
+                6: {
+
+                }
+            }
+        }
+    ],
+    "Firefly": [
+        {
+            name: "Firefly",
+            baseStats: {
+                Cooldown: "60s",
+            },
+            tiers: {
+                1: {
+
+                },
+                2: {
+
+                },
+                3: {
+
+                },
+                4: {
+
+                },
+                5: {
+
+                },
+                6: {
+
+                }
+            }
+        }
+    ],
+    "Seeker Mine": [
+        {
+            name: "Seeker Mine",
+            baseStats: {
+                Cooldown: "60s",
+            },
+            tiers: {
+                1: {
+
+                },
+                2: {
+
+                },
+                3: {
+
+                },
+                4: {
+
+                },
+                5: {
+
+                },
+                6: {
+
+                }
+            }
+        }
+    ],
+    "Drone": [
+        {
+            name: "Drone",
+            baseStats: {
+                Cooldown: "60s",
+            },
+            tiers: {
+                1: {
+
+                },
+                2: {
+
+                },
+                3: {
+
+                },
+                4: {
+
+                },
+                5: {
+
+                },
+                6: {
+
+                }
+            }
+        }
+    ],
+    "Shield": [
+        {
+            name: "Shield",
+            baseStats: {
+                Cooldown: "1s",
+            },
+            tiers: {
+                1: {
+
+                },
```

Comment:
Are these all partial data files or is this intentional? What does an empty tier mean?

Reply:
The point is to layout the structure. Like I was saying there’s a lot of data entry still left